theme

```diff
@@ -61,5 +61,6 @@
   "pbeckcom/green_slate_theme",
   "Poeschl/slate_red",
   "seangreen2/slate_theme",
-  "wowgamr/animated-weather-card"
+  "wowgamr/animated-weather-card",
+  "tgcowell/waves"
```

Comment:
Please keep the list sorted

Reply:
Hi @jbouwh apologies I really should have noticed that before submitting! I have updated this as per your advice.